Makefile

```
@@ -109,11 +54,14 @@ endif
 
 .PHONY: all modules clean help show check always-update install-models
 
-clean: # add more prerequisites for clean below
-	$(RM) -r $(SUB_VENV)
+clean: network-clean images-clean
 	$(RM) -r $(CURDIR)/venv # deliberately not using VIRTUAL_ENV here
 	$(RM) -r $(HOME)/.parallel/semaphores/id-ocrd_*
-	$(RM) ocrd-all-tool.json ocrd-all-module-dir.json ocrd-all-meta.json
+	$(RM) ocrd-all-tool.json ocrd-all-module-dir.json ocrd-all-meta.json ocrd-all-images.yaml
+
+.PHONY: images-clean
+images-clean:
```

```suggestion
.PHONY: clean-images
clean-images:
```
Maybe unimportant but this makes it more clear to me. Also the help-output would have to be adapted accordingly

---

Other clean subtargets use *-clean mnemonic, hence this choice

And I don't want clean images (as in "clean linen"), but the images to be cleaned...